src/main/java/com/ddudu/auth/jwt/converter/JwtConverter.java

```diff
@@ -0,0 +1,28 @@
+package com.ddudu.auth.jwt.converter;
+
+import com.ddudu.auth.jwt.JwtAuthToken;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.security.authentication.AbstractAuthenticationToken;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class JwtConverter implements Converter<Jwt, AbstractAuthenticationToken> {
+
+  private static final String USER_PRINCIPAL_NAME = "user";
+
+  private final JwtAuthorityConverter jwtAuthorityConverter = new JwtAuthorityConverter();
```

Comment:
```suggestion
  private final JwtAuthorityConverter jwtAuthorityConverter;
```
이런식으로 DI을 활용해도 될 것 같아요! 혹시 직접 new 키워드로 생성하신 이유가 있을까요?

Reply:
맞네요 이렇게 주입 했다가 기존 JwtAuthenticationConverter에서 하던 방식으로 따라한건데 주입 하겠습니다!